src/clean.rs

```diff
@@ -96,10 +97,10 @@ fn clean_aur(
     Ok(())
 }
 
-fn make_writeable(file: &Path) -> Result<()> {
+fn make_readable(file: &Path) -> Result<()> {
     let pkg = file.join("pkg");
     let mut perms = pkg.metadata()?.permissions();
-    perms.set_readonly(false);
+    perms.set_mode(0o755);
```

Comment:
Directories should be 644

Reply:
Files are 644, directories are 755 by default. The execute permission enables one to `cd` into that directory.

Although, it shouldn't make a difference either way since it will be removed right after.

Reply:
Yeah that's just me being dumb.